piksi_tools/console/update_view.py

```diff
@@ -565,30 +658,64 @@ def _compare_versions(self):
                 'serial_number'].value
         except:  # noqa
             pass
-        # Check if console is out of date and notify user if so.
         if self.prompt:
+            # Check if firmware is out of date and notify user if so.
+            remote_stm_version = self.newest_stm_vers
+
+            self.fw_outdated = remote_stm_version > local_stm_version
+            if local_stm_version.startswith('DEV'):
+                self.fw_outdated = False
+
+            if self.fw_outdated:
+                fw_update_prompt = \
+                    prompt.CallbackPrompt(
+                        title='Firmware Update',
+                        actions=[prompt.close_button]
+                    )
+
+                if 'fw' in self.update_dl.index[self.piksi_hw_rev]:
+                    fw_update_prompt.text = \
+                        "New Piksi firmware available.\n\n" + \
```

Comment:
Move to top-level multi-line string constant and use `str.format` instead of `%`?  See https://pyformat.info/ for more details.